e2e-tests/cypress/integration/do-you-want-to-add-another-comment.feature

```diff
@@ -1,4 +1,4 @@
-@testSuite
+@ignore
```

Comment:
same here

Reply:
since this page doesn't exist anymore I have just ignored this feature file for the time being instead of deleting them because we never know it might come back 😄